Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -471,5 +483,75 @@ private void FileList_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
                 ParentShellPageInstance.InteractionOperations.OpenSelectedItems(false);
             }
         }
+
+        private void QuickLook_Click(object sender, RoutedEventArgs e)
```

Comment:
@d2dyno1 To reduce duplicate code, couldn't we move these click event handlers into BaseLayout?

Reply:
@duke7553 I removed the handlers and bound the commands in https://github.com/files-community/Files/pull/4016/commits/b406f2507ef27055662eb65091c43227bd8e15dd

Reply:
The originally inteded way